test/functional/test_framework/messages.py

```diff
@@ -632,6 +633,9 @@ def set_null(self):
         self.hash = None
         self.scrypt256 = None
 
+    def set_base_version(self, n):
+        self.nVersion = n
```

Comment:
this sets the entire `nVersion` to `n`, not just the base? I assume the "base" is `nVersion & 0x000000ff`?

Reply:
Yeah, there's a follow-on PR that introduces chain IDs, which is the step after this.